app/worker/smtp.go

```diff
@@ -129,8 +130,12 @@ func processSMTPTask(smtpTask *model.SMTPTask, job *model.Job,
 		}
 	}
 
-	err = smtpClient.SendMail(smtpHostname, auth, from, recipients, msgBuffer.Bytes())
-	l.Debugf("processSMTPTask: smtpClient.SendMail returned %v", err)
+	fromAddress := getEmailAddress(from)
+	recipientAddresses := make([]string, len(recipients))
+	for i, recipient := range recipients {
+		recipientAddresses[i] = getEmailAddress(recipient)
+	}
+	err = smtpClient.SendMail(smtpHostname, auth, fromAddress, recipientAddresses, msgBuffer.Bytes())
```

Comment:
feel free to ignore: perhaps we should check if the `len(recipientAddresses)>0` ?

Reply:
https://github.com/mendersoftware/workflows/pull/169/commits/aeec87d906a92c8f840d8a627803640ed45e50c6